core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1179,8 +1180,18 @@ class KafkaApis(val requestChannel: RequestChannel,
       }
 
       def createResponse(requestThrottleMs: Int): AbstractResponse = {
+        def createFindCoordinatorResponse(error: Errors, node: Node): FindCoordinatorResponse = {
```

Comment:
Could probably modify `prepareResponse` to take the throttle time and remove this.

Reply:
The throttle time is only used in 2 places so I kept it this way. I'm happy to change it if you think that's really better

Reply:
No strong preference. Just a suggestion